.secrets.baseline

```diff
@@ -105,26 +105,145 @@
     },
     {
       "path": "detect_secrets.filters.heuristic.is_templated_secret"
-    },
-    {
-      "path": "detect_secrets.filters.regex.should_exclude_file",
-      "pattern": []
-    },
-    {
-      "path": "detect_secrets.filters.regex.should_exclude_secret",
-      "pattern": [
-        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9\\.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ\\..*",
-        "Lz57KV7UC1jY_zabLAiIkXkhktmV0Gwg7rG2Z05roY-Ow150MmfDWbVavcpGP8do88MebxM47H-0q30F-CfB2A",
-        "BiUZkuU4MFE8zg5zpghGdn-g-uepv14qXAXal4vpgnAk0qZSutsRFvhw_YNRoVwxsacBA6RCEHrHmYpTxsJ9sQ",
-        "diWgdrCGYnjrZK7cMPEKwJXvpGn6rvhCBteCl_I2ejg",
-        "ElcRbqkXk-XBnhYC55hApLY9oGnZA5H5MUlqe02gGXA",
-        "A84nU-ZLSFNs8VI6VXkunvWwRx-T3gAXvw2JPRrP78c",
-        "HMRCBearerToken",
-        "goodToken",
-        "badToken"
-      ]
     }
   ],
-  "results": {},
-  "generated_at": "2023-11-27T13:21:43Z"
+  "results": {
+    "infrastructure/public-api.yaml": [
+      {
+        "type": "JSON Web Token",
+        "filename": "infrastructure/public-api.yaml",
+        "hashed_secret": "d6b66ddd9ea7dbe760114bfe9a97352a5e139134",
+        "is_verified": false,
+        "line_number": 129
+      }
+    ],
+    "infrastructure/template.yaml": [
+      {
+        "type": "Secret Keyword",
+        "filename": "infrastructure/template.yaml",
+        "hashed_secret": "bc794cebe558155f13aba1743457f906cb8bd7a9",
+        "is_verified": false,
+        "line_number": 57
+      },
+      {
+        "type": "Secret Keyword",
+        "filename": "infrastructure/template.yaml",
+        "hashed_secret": "8a874a3fb21250a2518578da2c545eee007741ce",
+        "is_verified": false,
+        "line_number": 61
+      },
+      {
+        "type": "Secret Keyword",
+        "filename": "infrastructure/template.yaml",
+        "hashed_secret": "d5adc7795efc6aeb3901950a8dae86fd40f11de2",
+        "is_verified": false,
```

Comment:
We don't want to be doing this a better way is the code you've replaced on the left